e2e/utils/TestUtils.ts

```diff
@@ -16,7 +18,9 @@ export function createDefaultSettings() {
 }
 
 function createSettingsFile(settingsDir: string) {
-  const settingsData = '{}';
+  const settingsData = defaultSettings;
+
+  settingsData.pathManagementStrategy = PathManagementStrategy.RcFiles;
```

Comment:
Surprised we're using rcfiles instead of manual here (since this means we'll do things on developer machines when we run the tests locally).